program-runtime/src/invoke_context.rs

```diff
@@ -108,6 +108,67 @@ impl ComputeMeter {
     }
 }
 
+// bprumo TODO: doc where 128 GB came from. Move where the const is, also
+// 128 GB
+pub const DEFAULT_MAX_ACCOUNTS_DATA_LEN: usize = 128_000_000_000;
+
+/// bprumo TODO: doc, and/or move
+#[derive(Debug, Default, Clone, Copy, Eq, PartialEq)]
+pub struct AccountsDataBudget {
```

Comment:
Since this is local to a tx, could just store remaining

Reply:
Yes, I agree. I'm still prototyping the tracking of the accounts data len, so was trying to keep around more information for now. Once I have a good idea on how this all fits together I was planning to reduce the impl.